### PR TITLE
chore: update sqlchemy dependency

### LINKED ISSUE #5828: Depenendency Upgrades
The following dependencies have to be upgraded

- urllib3 = ">=1.24.2"
- SQLAlchemy = ">=1.3.0"
- Jinja2 = ">=2.10.1"
- marshmallow = ">=2.15.1"

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #5828

Updates the SQLAlchemy dependency to  `1.3.0`

### CODE REVIEW

**Comment by uds5501** [comment]
@iamareebjamal How do I resolve the lock conflicts?

**Comment by iamareebjamal** [comment]
git pull

File: requirements/common.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ raven[flask]
 healthcheck
 elasticsearch-dsl
 flask-redis
-SQLAlchemy==1.1.15 # 1.2.0 doesn't work, check if any future versions work
+SQLAlchemy>=1.3.0 # 1.2.0 doesn't work, check if any future versions work
```
Comment (iamareebjamal) [inline requirements/common.txt]:
```suggestion
SQLAlchemy>=1.3.0<2.0.0 # 1.2.0 doesn't work, check if any future versions work
```

**Comment by iamareebjamal** [comment]
Let the tests run first. They'll most likely fail

**Comment by uds5501** [comment]
@iamareebjamal Okay few queries, I tried `git pull upstream development`, no result.

And I have to use `pipenv install "SQLAlchemy>=1.3.0<2.0.0"`, correct?

**Comment by iamareebjamal** [comment]
Yes, the command is correct.

About `git pull upstream development`. That seems odd. Pipfile was updated in latest PR, it should be pulled

**Comment by uds5501** [comment]
@iamareebjamal This is the shell display on `git pull`

```
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=<remote>/<branch> dependency-update-sqlaclehmy

```

**Comment by iamareebjamal** [comment]
I meant `git pull upstream development`

**Comment by iamareebjamal** [comment]
Now your PR has conflicts in Pipfile.lock

You should have let the tests run first and if they passed, fix the conflicts

**Comment by uds5501** [comment]
@iamareebjamal I will move back one commit and push again, that will be fine?

**Comment by iamareebjamal** [comment]
Build is failing. Please check why

**Comment by uds5501** [comment]
@iamareebjamal This is happening due to : https://docs.sqlalchemy.org/en/13/changelog/migration_13.html#key-behavioral-changes-orm

As for this particular query, in `api/admin_sales/locations.py`,

```python
def sales_per_location_by_status(status):
    return db.session.query(
        Event.location_name.label('location'),
        func.sum(Order.amount).label(status + '_sales'),
        func.sum(OrderTicket.quantity).label(status + '_tickets')) \
                     .outerjoin(Order) \
                     .outerjoin(OrderTicket) \
                     .filter(Event.id == Order.event_id) \
                     .filter(Order.status == status) \
                     .group_by(Event.location_name, Order.status) \
                     .cte()
```
is the issue.

I can change ` .outerjoin(Order)` to `.outerjoin(Order, Order.event_id==Event.id)` but what do I do about `.outerjoin(OrderTicket)`?
Shall I use `.outerjoin(OrderTicket, OrderTicket.order_id==Order.id)`?

**Comment by iamareebjamal** [comment]
@uds5501 See the previous result and match it after upgrading

**Comment by uds5501** [comment]
@iamareebjamal have a look?

**Comment by iamareebjamal** [comment]
Great. Please resolve conflicts and I will merge it. Also, remove the comment about SQLAlchemy in requirements.txt

**Pull request merged**